dockerfiles/examples/nfs/hpcc-nfs/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: hpcc-nfs
+description: A Helm chart to provision HPCC PVC's using an NFS deployment as shared storage
+
+# A chart can be either an 'application' or a 'library' chart.
```

Comment:
Not sure these comments are useful

Reply:
Agree.
These are a result of helm create auto-generation (as is .helmignore), I trimmed some of the excess, but not sure where to stop.
e.g. should all the unused definitions in _helpers.tpl be removed too?

Reply:
These comments are notes to the developer writing a new chart, and serve no purpose once you have done so.

Yes, you probably should trim unused definitions in helpers too